src/wallet/wallet.cpp

```diff
@@ -87,7 +87,7 @@ const CWalletTx* CWallet::GetWalletTx(const uint256& hash) const
     return &(it->second);
 }
 
-CPubKey CWallet::GenerateNewKey()
+CPubKey CWallet::GenerateNewKey(bool internal)
```

Comment:
IMO it would be better to use an enum for internal vs external rather than a bool.